packages/intl/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@4c/intl-cli",
```

Comment:
I wanted to save `@4c/intl` it sounds more like a library we might make idk

Reply:
seems reasonable for `@4c/intl` to also ship the CLI though :p e.g. doesn't react-intl ship both the components and the babel xform?

Reply:
sure i just wasn't sure that that package would make sense with this cli. IDK i'm fine with whatever, it was more that `@4c/intl` felt like a good name i didn't want to waste on a json concatenater

Reply:
you can always cut a breaking change later :p

Reply:
fair